.pre-commit-config.yaml

```diff
@@ -48,6 +48,7 @@ repos:
             tools/clean.py|
             tools/copyright.py|
             tools/dht_flood.py|
+            tools/libtorrent_lldb.py|
```

Comment:
This shouldn't be masked

It looks like `tools/libtorrent_lldb.py` is auto-formatted and linted in `master`, but not in `RC_2_0`.